rp_asos/tasks.py

```diff
@@ -292,3 +294,50 @@ def run(self, project_id, tz_code, **kwargs):
 
 
 create_hospital_groups = CreateHospitalGroups()
+
+
+class ScreeningRecordCheck(Task):
+    """
+    Checks all the active screening records linked to the organisation and
+    notifies the steering group of records that have not been updated in the
+    last 3 days.
+    """
+
+    name = "rp_asos.tasks.screening_record_check"
+    log = get_task_logger(__name__)
+
+    def run(self, org_id):
+        org = Organization.objects.prefetch_related(
+            "projects",
+            "projects__hospitals",
+            "projects__hospitals__screening_records",
+        ).get(id=org_id)
+
+        no_update = []
```

Comment:
I think we might want to rename this to something like `list_of_hospitals_without_update`. It's current name feels a bit like a boolean value